language-service/language_service/dto/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+These objects are what will be used to respond to clients.
+"""
+
+class Definition:
+    def __init__(self, subdefinitions=[], tag="", examples=[], optional_fields=[]):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)